python/ray/_private/runtime_env.py

```diff
@@ -113,17 +150,15 @@ def __init__(self, runtime_env_json: dict):
                 raise TypeError("runtime_env['pip'] must be of type str or "
                                 "List[str]")
 
-        if "working_dir" in runtime_env_json:
-            self._dict["working_dir"] = runtime_env_json["working_dir"]
-        else:
-            self._dict["working_dir"] = None
-
         if "uris" in runtime_env_json:
             self._dict["uris"] = runtime_env_json["uris"]
 
         if "_ray_release" in runtime_env_json:
             self._dict["_ray_release"] = runtime_env_json["_ray_release"]
```

Comment:
We don't need this since we have already get `_ray_commit`

Reply:
Sounds good, I will remove it

Reply:
I added it back for now just in case there's some backward compatibility issues.  We can always remove it later